index.d.ts

```diff
@@ -118,12 +118,10 @@ export type CommonOptions<EncodingType extends EncodingOption = DefaultEncodingO
 	- `'inherit'`: Re-use the current process' `stdin`.
 	- an integer: Re-use a specific file descriptor from the current process.
 	- a Node.js `Readable` stream.
-
-	Unless the synchronous methods are used, the value can also be a:
-	- file path. If relative, it must start with `.`.
-	- file URL.
```

Comment:
This was a mistake: file paths and URLs are actually allowed with the synchronous methods.